### PR TITLE
feat: about page

### PR DESCRIPTION
**Pull request by 1MrSmith**

### CODE REVIEW

**Comment by 1MrSmith** [comment]
[Landing Page (About page)](https://trello.com/c/OakMc1vX/11-landing-page-about-page)

File: js_machine_front/src/components/TileContent.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+export const TileContent: React.FC<Props> = memo((props: Props) => {
+  return (
+    <div className="content color">
+      <div className="container">
+        <div className="container_flex-container">
+          {(() => {
+            const tiles = [];
+
+            for (let i = 0; i < 3; i++) {
```
Comment (vharadkou) [inline js_machine_front/src/components/TileContent.tsx]: use arrTiles.map instead of for

**Review by vharadkou — CHANGES_REQUESTED** [review]

File: js_machine_front/src/components/TileContent.tsx (hunk 1 above)
Comment (1MrSmith) [reply js_machine_front/src/components/TileContent.tsx]: Ok

Comment (1MrSmith) [reply js_machine_front/src/components/TileContent.tsx]: fix

File: js_machine_front/src/components/TileContent.tsx [hunk 2]
```diff
@@ -11,20 +11,13 @@ export const TileContent: React.FC<Props> = memo((props: Props) => {
...
-                  title={props.arrTiles[i].title}
-                />,
-              );
-            }
-
-            return tiles;
-          })()}
+          {props.arrTiles.map((item: Tile, i: number) => (
+            <TileElement
+              key={i}
```
Comment (vharadkou) [inline js_machine_front/src/components/TileContent.tsx]: don't use index as a key, put some id

**Review by vharadkou — CHANGES_REQUESTED** [review]

File: js_machine_front/src/components/TileContent.tsx (hunk 2 above)
Comment (1MrSmith) [reply js_machine_front/src/components/TileContent.tsx]: fix

**Review by vharadkou — APPROVED** [review]

**Pull request merged**